hztb-servicemanager/src/main/java/com/svs/hztb/service/impl/GroupDataServiceImpl.java

```diff
@@ -33,72 +32,81 @@
 
 	@Autowired
 	GroupRepository groupRepository;
-	
+
 	@Autowired
 	UserGroupEntityRepository userGroupEntityRepository;
-	
+
 	@Autowired
 	UserRepository userRepository;
-	
+
 	@Override
 	public GroupOutput removeGroup(GroupInput groupInput) {
-		
+
 		GroupOutput groupOutput = null;
 		try {
 			groupRepository.setGroupNameFor(DEFAULT_GROUP_NAME, groupInput.getGroupId(), groupInput.getUserId());
 			groupOutput = buildGroupOutput();
-		} catch(Exception e) {
-			
+		} catch (Exception e) {
+
 		}
 		return groupOutput;
-		
+
 	}
-	
+
 	@Override
 	public GroupOutput createGroup(GroupInput groupInput) {
 		GroupOutput groupOutput = null;
 		try {
-			//Create group
+			// Create group
 			GroupEntity groupEntity = FunctionUtils.convert(groupInput, new GroupInputToEntityConverter());
 			groupRepository.saveAndFlush(groupEntity);
-		//	groupRepository.findByGroupName(groupEntity.getGroupName());
-		
-			//Add group members
+			// groupRepository.findByGroupName(groupEntity.getGroupName());
+
+			// Add group members
 			List<UserGroupEntity> userGroupList = createUserGroup(groupInput.getAddMembers(), groupEntity);
 			userGroupEntityRepository.save(userGroupList);
 			groupOutput = buildGroupOutput();
-		
-		} catch(Exception e) {
-			
+			GroupDetail groupDetail = new GroupDetail();
+			List<GroupDetail> groupDetailList = new ArrayList<GroupDetail>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS2293)